examples/bert/__init__.py

```diff
@@ -1,2 +1,70 @@
 # Copyright 2022 MosaicML Examples authors
 # SPDX-License-Identifier: Apache-2.0
+
+try:
+    import torch
+
+    # yapf: disable
+    from examples.bert.src.bert_layers import (BertEmbeddings, BertEncoder,
+                                               BertForMaskedLM,
+                                               BertForSequenceClassification,
+                                               BertGatedLinearUnitMLP,
+                                               BertLayer, BertLMPredictionHead,
+                                               BertModel, BertOnlyMLMHead,
+                                               BertOnlyNSPHead, BertPooler,
+                                               BertPredictionHeadTransform,
+                                               BertSelfOutput,
+                                               BertUnpadAttention,
+                                               BertUnpadSelfAttention)
+    # yapf: enable
+    from examples.bert.src.bert_padding import (IndexFirstAxis,
+                                                IndexPutFirstAxis,
+                                                index_first_axis,
+                                                index_put_first_axis, pad_input,
+                                                unpad_input, unpad_input_only)
+    if torch.cuda.is_available():
+        from examples.bert.src.flash_attn_triton import \
+            flash_attn_func as flash_attn_func_bert # type: ignore
+        from examples.bert.src.flash_attn_triton import \
+            flash_attn_qkvpacked_func as flash_attn_qkvpacked_func_bert # type: ignore
+    from examples.bert.src.hf_bert import (create_hf_bert_classification,
+                                           create_hf_bert_mlm)
+    from examples.bert.src.mosaic_bert import (
+        create_mosaic_bert_classification, create_mosaic_bert_mlm)
+except ImportError as e:
+    raise Exception(
```

Comment:
My thinking was that it would be nice to show both 1) the helpful message that I've added, and 2) the specific import that failed (will only show the first one, but still helpful). Open to changing though

Reply:
As for the `ImportError`, yeah, I'll change those over.